draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3878,6 +3902,25 @@ RESPONSE:
 
 	<section title="IANA Considerations" anchor="sect-7">
    <section title="CDNI Payload Type Parameter Registrations" anchor="sect-7.1"><t>
+
```

Comment:
I debated putting this before the rest of 11.1 versus making a new section for it altogether, but given that everything pertained to "CDNI Payload Type Parameter Registrations", I came to the conclusion that making this part of 11.1 was the better approach.  I don't feel strongly about it, so if you disagree, please let me know.